src/Microsoft.ML.TimeSeries/SequentialAnomalyDetectionTransformBase.cs

```diff
@@ -0,0 +1,655 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.CpuMath;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+
+namespace Microsoft.ML.Runtime.TimeSeriesProcessing
+{
+    // REVIEW saamizad: This base class and its children classes generate one output column of type VBuffer<Double> to output 3 different anomaly scores as well as
```

Comment:
>saamizad [](start = 14, length = 8)

remove aliases

Reply:
Keep the 'REVIEW' though

---
In reply to: [219996812](https://github.com/dotnet/machinelearning/pull/977#discussion_r219996812) [](ancestors = 219996812)